packages/config/config/devices/0x010f/fgd212_3.5.json

```
@@ -57,37 +57,39 @@
 		},
 		"4": {
 			"label": "Incandescence time of dimmable CFL*",
-			"description": "time required for switching. 0s - 25,5s",
+			"description": "Each step is 100 ms, e.g. 5 = 500 ms",
```

This should be a unit instead.
```suggestion
			"unit": "100 ms",
```

---

Thanks, reading carefully I now see that this is also mentioned in the style guide. I probably missed this caused there was no example of it there.